index.html

```diff
@@ -6347,7 +6355,7 @@ <h5>Presentational Role Inheritance</h5>
   &lt;foo&gt; Sample Content &lt;/foo&gt;
   &lt;foo&gt; More Sample Content &lt;/foo&gt;
 &lt;/foo&gt;</pre>
-				<p class="note">There are other <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> roles with required children for which this situation is applicable (e.g., radiogroups and listboxes), but tables and lists are the most common real-world cases in which the presentation inheritance is likely to apply.</p>
+				<p class="note">There are other <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> roles with specific allowed children for which this situation is applicable (e.g., radiogroups and listboxes), but tables and lists are the most common real-world cases in which the presentation inheritance is likely to apply.</p>
```

Comment:
See my comment on radiogroups. Don't think this should have allowed child roles.

```suggestion
				<p class="note">There are other <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> roles with specific allowed children for which this situation is applicable (e.g., feeds and listboxes), but tables and lists are the most common real-world cases in which the presentation inheritance is likely to apply.</p>
```